src/Impostor.Api/Net/Inner/Objects/ITaskInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Impostor.Api.Net.Messages;
+
+namespace Impostor.Api.Net.Inner.Objects
+{
+    public interface ITaskInfo
+    {
+        uint Id { get; }
+        
+        bool Complete { get; }
+
+        void Serialize(IMessageWriter writer);
```

Comment:
I don't think `Serialize` and `Deserialize` should be exposed to the plugin API.

Reply:
Resolved in 312fa43bac9a6538bc99c9d087f1f5ab01308031